src/vis/vis.js

```diff
@@ -744,6 +744,11 @@ var Vis = View.extend({
       }
     }
 
+    vizjson.overlays.push({
+      type: 'attribution',
+      order: 5
```

Comment:
Good question, I feel it is related to the position in the "map grid", but @javierarce knows more about it. In any case I'm going to remove it because it is always displayed in the same place.